Doberman/AlarmMonitor.py

```diff
@@ -230,11 +230,14 @@ def check_shifters(self):
         now = dtnow()
         shift = self.db.read_from_db('settings', 'shifts',
                 {'start': {'$lte': now}, 'end': {'$gte': now}}, onlyone=True)
+        if shift == None:
+            self.current_shifters = []
+            return
         new_shifters = shift['shifters']
         new_shifters.sort()
         shift_end = shift['end']
         if new_shifters != self.current_shifters and len(''.join(new_shifters)) != 0:
```

Comment:
There should be an alarm to the old shifters if there are no new shifters

Reply:
Oops, maybe this is not the right place for this comment :D

Reply:
The comment is valid but indeed not really related to this PR. Probably better to open an issue.